### PR TITLE
Set storage policy action

### PR DESCRIPTION
**Pull request by timmyyao**

### CODE REVIEW

**Comment by drankye** [comment]
Looks like the useful tests for mover tool will be removed. Will we bring them back later?

File: smart-actions/src/main/java/org/smartdata/actions/hdfs/SetStoragePolicyAction.java [hunk 1]
```diff
@@ -1,17 +1,65 @@
...
+
+  private String fileName;
+  private String storagePolicy;
+  private String name = "SetStoragePolicyAction";
+
+  public SetStoragePolicyAction() {
+    this.setActionStatus(new ActionStatus());
+  }
+
+  public String getName() {
```
Comment (drankye) [inline smart-actions/src/main/java/org/smartdata/actions/hdfs/SetStoragePolicyAction.java]: Will all action have a name now kept in the instance? If so, any different from the name associated with action factory map?

**Comment by drankye** [comment]
Tim, please rebase this, thanks.

**Comment by timmyyao** [comment]
This PR is already included in another PR.

**Pull request closed without merging**